static/docs/user-guide/usability.md

```diff
@@ -0,0 +1,141 @@
+# Usability
+
+You can enrich your DVC experience with the following:
+
+## Shell command completion
+
+Command completion is usually requested by pressing the `tab` key on your
+shell, it will then present the possible options that could follow that command
+call. For example:
+
+```bash
+dvc r # Press [tab] key
```

Comment:
we have a convention to do 4 spaces indent in code blocks and start commands with $: `    $ dvc init`